### ISSUE TITLE
Unable to configure windows instance

### ISSUE THREAD

**Issue by Nucs** [nylonee/watchlistarr#109]
Hey, using the latest **0.21 release**
I have attempted most variations from the closed/open issues and in documentation to set sonarr and radarr configuration.
I have:
1. Set environment variables on the machine
![image](https://github.com/nylonee/watchlistarr/assets/649919/bb553a19-af91-42cb-b62f-da9685ea6aef)
2. I have changed the generated config file
![image](https://github.com/nylonee/watchlistarr/assets/649919/bf34a6ce-bd59-4523-a33b-dfb7c95c1686)
3. I have added env variables prior to calling the java jar command
```
set SONARR_BASE_URL=127.0.0.1:45261
set RADARR_BASE_URL=127.0.0.1:45263
set interval_seconds=10
...
```
4. Have have added configs to the java jar command
`java -Dsonarr.apikey=X -Dradarr.apikey=X -Dplex.token=X -Dsonarr.baseUrl="http://127.0.0.1:45261" -Dradarr.baseUrl="http://127.0.0.1:45263" -Dsonarr.qualityProfile="WEB-DL (1080p)" -Dradarr.qualityProfile="HD Bluray + WEB" -Dsonarr.rootFolder="C:/state/plex/Sonarr/bin" -Dradarr.rootFolder="C:/state/plex/Radarr/bin" -Dsonarr.seasonMonitoring=all -Dinterval.seconds=10 -jar watchlistarr.jar`

---
None of the above have worked and I end up with this error and callstack:
```
java.lang.IllegalArgumentException: Unable to connect to Sonarr at http://localhost:8989, with error java.io.IOException: The remote computer refused the network connection.

        at configuration.ConfigurationUtils$.throwError(ConfigurationUtils.scala:263)
        at configuration.ConfigurationUtils$.$anonfun$getSonarrConfig$4(ConfigurationUtils.scala:93)
        at getAndSet @ org.typelevel.keypool.KeyPool$.destroy(KeyPool.scala:120)
        at flatMap @ org.typelevel.keypool.KeyPool$.destroy(KeyPool.scala:120)
        at main$ @ Server$.main(Server.scala:11)
        at getAndSet @ org.typelevel.keypool.KeyPool$.destroy(KeyPool.scala:120)
        at clear @ org.http4s.client.middleware.FollowRedirect$.redirectLoop$1(FollowRedirect.scala:105)
        at clear @ org.http4s.client.middleware.FollowRedirect$.redirectLoop$1(FollowRedirect.scala:105)
```

**To summarize:**
I can't configure the app properly.
It ignored the values from all the locations I've set.
Also how do I specify the config file, "-Dconfig=config/config.yaml" will work? there is lacking documentation in this area

Thanks!

**Comment by Nucs** [comment]
Related issues
#107
#105

**Comment by nylonee** [comment]
Thanks for raising this, apologies for taking so long to get back to you. Were you able to resolve this?
I'll be improving the documentation today, so hopefully there will be some insight into what your issue is

**Comment by nylonee** [comment]
Also could you enable debug mode and let me know if there's any more useful logs?

**Comment by nylonee** [comment]
Ah this is a bit counterintuitive (and I'll push a fix now), but I just tried a similar config locally and what happens for me is the IP address can't resolve to a URL host, so it falls back to trying the default (but it doesn't tell you anything useful in the logs)

The fix for now is to do:
```
http://127.0.0.1:45261
```

Also when you're providing Java properties, you need to enclose the full property in quotation marks (literally don't ask me why):
```
"-Dsonarr.baseUrl=http://127.0.0.1:45261"
```
